database/postgres/dml/repo.go

```diff
@@ -21,6 +21,16 @@ WHERE user_id = ?
 ORDER BY id DESC
 LIMIT ?
 OFFSET ?;
+`
+
+	// ListPrivateOrgRepos represents a query to list
+	// all repos for an org in the database.
+	ListPrivateOrgRepos = `
+SELECT *
+FROM repos
+WHERE org = ?
+AND visibility = 'private'
+ORDER BY id DESC;
```

Comment:
I think we're missing `LIMIT` and `OFFSET` from this query.

AFAICT, this would capture every repo in the given org with a visibility of `private`.

This can cause a problem if the number of repos returned by the query is abnoramlly large.

You can use the below for reference:

https://github.com/go-vela/server/blob/fefd9eec0fbad52a17717104b23375a0bcf075e9/database/postgres/dml/repo.go#L15-L23